packages/webapp/pages/onboarding.tsx

```diff
@@ -176,16 +177,31 @@ export function OnboardPage(): ReactElement {
   const [isPlusCheckout, setIsPlusCheckout] = useState(false);
   const hasSelectTopics = !!feedSettings?.includeTags?.length;
 
+  const isReorderExperiment = useFeature(featureOnboardingReorder);
```

Comment:
Should we maybe use a conditional feature and check if `isOnboardingReady` and that the user is not logged in.

Reply:
useConditionalFeature works only for  logged in users, I could change it to something like this in order to avoid enrollment for logged users (maybe with useMemo):

```ts
const isReorderExperiment = isOnboardingReady && !user && getFeatureValue(featureOnboardingReorder)
```

wdyt?

Reply:
Why is it restricted for anonymous users? Yeah, that snippet can work, but curious why useConditionalFeature wouldn't work on anon users.

Reply:
NVM I see that now the code for useConditionalFeature has been changed, I'll try to use it 👀

Reply:
I think that we should not enroll users that are logged in and in tag selection step, wdyt?

Reply:
Yes, that was my main concern actually. But since we check that the user is not logged in, we should be safe from that since only logged-in users can reach that.